public/locales/ru/translations.json

```diff
@@ -2,52 +2,52 @@
-  "recent": "Недавняя активность",
-  "no followers": "Нет подписчиков",
-  "latest followers": "Последние подписчики",
+  "recent": "Недавняя Активность",
+  "no followers": "Ни одного подписчика",
+  "latest followers": "Последние Подписавшиеся",
   "connect wallet to view": "Подключите кошелек для просмотра",
   "phrase": "Все дело в том, кого ты знаешь. 😜",
   "social graph first": "Социальный граф",
-  "social graph second": "для Ethereum.",
+  "social graph second": "для сети Ethereum.",
   "follow ghost first": "Следите за друзьями.",
   "follow ghost second": "Избегайте врагов.",
   "follower list first": "Список подписок",
   "follower list second": "которым вы владеете.",
   "integrations": "Интеграции",
   "partners": "Партнеры запуска",
   "sponsors": "Поддерживается грантами от:",
-  "support": "Поддержка",
+  "support": "Служба Поддержки",
 
   "profile": "Мой Профиль",
   "profile error": "Пользователя не существует",
   "settings": "Настройки Списка",
   "block-mute": "Списки Блок./Игнор.",
   "edit settings": "Изменить Настройки",
-  "edit profile": "Изменить профиль",
-  "top eight title": "🔥 Мой Топ 8 🔥",
-  "no top eight": "Здесь ничего нет... пока еще 😳",
-  "top eight limit": "Ваш Топ 8 заполнен! Чтобы добавить нового человека, сперва удалите кого-то. 😈",
-  "top eight description": "Добавляйте друзей в свой Топ 8! Вы всегда сможете добавить или удалить людей позже.",
+  "edit profile": "Редактировать",
+  "top eight title": "🔥 Мой Топ-8 🔥",
+  "no top eight": "Здесь ничего нет... пока что 😳",
+  "top eight limit": "Ваш Топ-8 заполнен! Чтобы добавить нового человека, сперва удалите кого-то. 😈",
+  "top eight description": "Добавляйте друзей в свой Топ-8! Вы всегда сможете добавить или удалить людей позже.",
   "apply": "Применить",
   "edit": "Редактировать",
-  "no bio": "Информация о себе не заполнена",
+  "no bio": "Нет данных профиля",
   "copied": "Скопировано",
-  "add to top eight": "Добавить в 🔥Мой Топ 8🔥",
-  "remove from top eight": "Удалить из 🔥Моего Топ 8🔥",
+  "add to top eight": "Добавить в 🔥Мой Топ-8🔥",
+  "remove from top eight": "Удалить из 🔥Моего Топ-8🔥",
   "copy address": "Копировать Адрес",
-  "copy ens": "Копировать ENS",
+  "copy ens": "Копировать имя ENS",
   "copy profile": "Копировать Ссылку На Профиль",
   "tags": "Теги",
   "no tag": "Нет тега",
   "no tags": "Нет тегов",
   "follower count": "По числу подписчиков",
-  "latest first": "Сначала новые",
-  "earliest first": "Сначала старые",
+  "latest first": "Сначала Новые",
+  "earliest first": "Сначала Старые",
   "not primary list": "Не подтверждено пользователем",
-  "not primary list tooltip": "Этот список не подтвержден как принадлежащий указанному пользователю. Пользователь должен установить список как Основной, чтобы подтвердить владение ним.",
+  "not primary list tooltip": "Этот список не подтвержден как принадлежащий указанному пользователю. Пользователь должен сделать список Основным, чтобы подтвердить владение ним.",
```

Comment:
Здесь и далее используется «сделать список Основным» вместо «установить список как Основной», так как второй вариант является калькой с английского языка и не настолько же часто используется в обиходе как в первый.

From here on we use «сделать список Основным» (make list Primary) instead of «установить список как Основной» (set list as Primary) because the second option is a tracing from English and is not used as often in everyday life as the first.

Reply:
Придерживался оригинала, можем поменять. 
✅

Reply:
👍